things/postgres/channels_test.go

```
@@ -201,7 +201,7 @@ func TestMultiChannelRetrieval(t *testing.T) {
 	}
 
 	for desc, tc := range cases {
-		page := chanRepo.RetrieveAll(tc.owner, tc.offset, tc.limit)
+		page, _ := chanRepo.RetrieveAll(tc.owner, tc.offset, tc.limit)
```

Don't ignore errors, assert error too.